Shared/Samples/Animate images with image overlay/AnimateImagesWithImageOverlayView.swift

```diff
@@ -0,0 +1,174 @@
+    
+    /// The text for the start stop button.
+    @State private var startStopButtonText = "Start"
+    
+    var body: some View {
+        VStack {
+            // Create a scene view to display the scene.
+            SceneView(scene: model.scene, imageOverlays: [model.imageOverlay])
+                .onAppear {
+                    // Create display link and load first image.
+                    model.displayLink = model.makeDisplayLink()
+                    model.setImageFrame()
+                }
+                .onDisappear {
+                    // Invalidate display link before exiting.
+                    model.displayLink.invalidate()
+                }
+            VStack {
+                HStack {
+                    Slider(value: $model.imageOverlay.opacity, in: 0.0...1.0, step: 0.01)
+                    VStack {
+                        Text("Opacity")
+                        Text(model.imageOverlay.opacity, format: .percent.precision(.fractionLength(0)))
+                    }
+                }
+                .padding([.top, .horizontal])
+                HStack {
+                    Button(startStopButtonText) {
+                        startStopButtonText = model.displayLink.isPaused ? "Stop" : "Start"
+                        model.displayLink.isPaused.toggle()
+                    }
+                    Spacer()
+                    Menu("Speed") {
+                        Picker("Speed Selection", selection: $model.displayLink.preferredFramesPerSecond) {
+                            Text("Slow").tag(15)
+                            Text("Medium").tag(30)
+                            Text("Fast").tag(60)
+                        }
+                    }
+                }
+                .padding()
+            }
+        }
+    }
+}
+
+private extension AnimateImagesWithImageOverlayView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A scene with a dark gray base and centered on Southern California.
+        let scene: ArcGIS.Scene = {
+            // Creates a scene and set an initial viewpoint.
+            let scene = Scene(basemapStyle: .arcGISDarkGrayBase)
+            let point = Point(x: -116.621, y: 24.7773, z: 856977.0, spatialReference: .wgs84)
+            let camera = Camera(location: point, heading: 353.994, pitch: 48.5495, roll: 0)
+            scene.initialViewpoint = Viewpoint(latitude: .nan, longitude: .nan, scale: .nan, camera: camera)
+            
+            // Add base surface from elevation service.
+            let elevationSource = ArcGISTiledElevationSource(url: .worldElevationService)
+            let surface = Surface()
+            surface.addElevationSource(elevationSource)
+            scene.baseSurface = surface
+            return scene
+        }()
+        
+        /// A timer to synchronize image overlay animation to the refresh rate of the display.
+        @Published var displayLink: CADisplayLink!
```

Comment:
We could just give it a default value instead of implicitly unwrapping it like so:
```suggestion
        @Published var displayLink = CADisplayLink()
```

But I am not sure which way is really better for this situation so I'll let @yo1995 comment if he disagrees.